src/Control/Carrier/Pure.hs

```diff
@@ -1,62 +0,0 @@
-{-# LANGUAGE EmptyCase, MultiParamTypeClasses #-}
-
--- | A carrier for pure effects, used to kick off a stack of effects with 'run'.
---
--- @since 1.0.0.0
-module Control.Carrier.Pure
```

Comment:
When I added the `Algebra` instance for `Identity`, @patrickt asked if we should eliminate `PureC` since it’s now redundant. At the time I felt like it would be better to keep our own definition around and define our own instances, but upon reflection this only makes interoperation with other libraries harder since it’s probable that our users are using `run` instead of `runIdentity` as the terminus of pure effect stacks. By eliminating `PureC`, and redefining `run` as a synonym for `runIdentity`, we not only reduce our surface area, but also operate seamlessly with anything _else_ defining instances for `Identity`.

Reply:
Well-reasoned.